tests/PartialCommonOwnership721/tests.ts

```diff
@@ -354,6 +355,7 @@ async function tests(config: TestConfiguration): Promise<void> {
     beneficiary = new Wallet(contract, signers[1]);
     alice = new Wallet(contract, signers[2]);
     bob = new Wallet(contract, signers[3]);
+    deployer = new Wallet(contract, signers[9]);
```

Comment:
Why the addition?

Reply:
Silly remainder from development, removed